src/app/map/[id]/components/Map.tsx

```diff
@@ -632,7 +632,7 @@ export default function Map({
           </>
         )}
       </MapGL>
-      <div className="absolute top-4 right-4 z-20">
+      <div className="fixed z-100 top-[10px] right-4">
```

Comment:
request: can we move the search box into the PrivateMapNavbar component, instead of keeping it in the Map component?